ietf/utils/draft.py

```diff
@@ -630,7 +630,7 @@ def extract_authors(self):
         address_section = r"^ *([0-9]+\.)? *(Author|Editor)('s|s'|s|\(s\)) (Address|Addresses|Information)"
 
         ignore = [
-            "Standards Track", "Current Practice", "Internet Draft", "Working Group",
+            "Standards Track", "Current Practice", "Internet-Draft", "Working Group",
```

Comment:
I need to look deeper, but I think this is looking for text produced by another program?

Reply:
`xml2rfc` always produce `Internet-Draft`.